src/style.css

```diff
@@ -2,0 +1,94 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
```

Comment:
As with the previous `@font-face`, the `font-family` should not include multiple font faces. Declare the font family once and use `font-weight` to differentiate between different weights.